platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ProjectColorTabDecorator.java

```diff
@@ -161,8 +161,16 @@ public void paintAfter( TabData tab, Graphics g, Rectangle tabRect, boolean isSe
         }
         g.setColor( c );
         Rectangle rect = new Rectangle( tabRect );
-        rect.y += rect.height - 3;
-        rect.grow( -1, -1 );
+        int underlineHeight = UIManager.getInt("nb.multitabs.underlineHeight"); // NOI18N
+        if( underlineHeight > 0 ) {
+            // if the selected tab is highlighted with an "underline" (e.g. in FlatLaf)
+            // then paint the project color bar at the top of the tab
+            rect.height = underlineHeight;
```

Comment:
If an underline is used to indicate the active tab, why not just keep the whole tab colored, like the other tabs, with the underline added? E.g.

rect.height - =underlineHeight?

Reply:
@eirikbakke That would look better. Thanks Eirik!